village/src/components/SingleSmurf.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import Styled from 'styled-components'
+import { NavLink } from 'react-router-dom'
+import UpdateSmurf from './UpdateSmurf'
+
+const Div = Styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 20px;
+    padding: 20px;
+    width: 300px;
+    height: 300px;
+    text-align: center;
+    border: solid black;
+    border-radius: 50%;
+    background: dodgerblue;
+    color: white;
+`
+const Button = Styled.button`
+    -moz-box-shadow: 0px 1px 0px 0px #f0f7fa;
```

Comment:
Styled Components automatically "vendor-prefixes" your styles, so you don't need to do it manually add the styles for the different browsers.